integration_tests/suite/test_subscriptions.py

```diff
@@ -217,24 +225,30 @@ def test_given_wrong_auth_when_create_subscription_then_401(self):
                     raises(WebhookdError, has_property('status_code', 401)))
 
     def test_when_create_invalid_subscription_then_400(self):
-        webhookd = self.make_webhookd(VALID_TOKEN)
+        webhookd = self.make_webhookd(MASTER_TOKEN)
 
         assert_that(calling(webhookd.subscriptions.create).with_args(INVALID_SUBSCRIPTION),
                     raises(WebhookdError, has_property('status_code', 400)))
 
     def test_when_create_http_subscription_then_subscription_no_error(self):
-        webhookd = self.make_webhookd(VALID_TOKEN)
+        webhookd = self.make_webhookd(MASTER_TOKEN)
 
         response = webhookd.subscriptions.create(TEST_SUBSCRIPTION)
         subscription_uuid = response['uuid']
 
+        assert_that(response, has_key('owner_tenant_uuid'))
+        assert_that(response['owner_tenant_uuid'], MASTER_TENANT)
```

Comment:
You can use `assert_that(response, has_entries(owner_tenant_uuid=MASTER_TENANT))` instead of checking the key and after checking the value

Reply:
done

Reply:
An now you can delete `assert_that(response, has_key('owner_tenant_uuid'))` :)